src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne || '0');
+  const two = Big(numberTwo || (operation === '÷' || operation === 'x' ? '1' : '0'));
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
+  if (operation === '%') {
+    return one.mod(two).toString();
```

Comment:
- [ ]  This is a modulus operation, but I don't know why your app is not behaving accordingly. When I click a number and the `%` operator it gives a fraction of the number divided by 100. E.g:- If I click `8` and `%` it displays `0.08` in the input field. Kindly fix this bug. But you shouldn't touch this file. You can fix this bug in your components.